andrey_n/homework_5/homework_5/tests/test_main.py

```diff
@@ -0,0 +1,115 @@
+import pytest
+
+import main
+
+
+@pytest.mark.parametrize(
+    "input_data,expected",
+    [(10, 20), ([1, 2], [1, 2, 1, 2])],
+)
+def test_double_input(input_data, expected):
+    """Test double_input function"""
+    assert main.double_input(input_data) == expected
+
+
+def test_double_input_raises():
+    """Test double_input function raises exception"""
+    with pytest.raises(TypeError):
+        main.double_input(list)
+
+
+@pytest.mark.parametrize(
+    "input_data,mask,expected",
+    [
+        (
+            "apples, pears # and bananas\ngrapes\nbananas !apples",
+            ["#", "!"],
+            "apples, pears\ngrapes\nbananas",
+        ),
+        (
+            "apples, pears # and bananas\nbananas ? peach \nbananas ?apples",
+            ["#", "?"],
+            "apples, pears\nbananas\nbananas",
+        ),
+    ],
+)
+def test_strip_comments(input_data, mask, expected):
+    """Test strip_comments function"""
+    assert main.strip_comments(input_data, mask) == expected
+
+
+@pytest.mark.parametrize(
+    "input_data,mask,expected",
+    [
+        (
+            "apples, pears # and bananas\ngrapes\nbananas !apples",
+            ["#", "!"],
+            "apples, \ngrapes\nbananas",
+        ),
+        (
+            "apples, pears # and bananas\nbananas ? peach \nbananas ?apples",
+            ["#", "?"],
+            "apples, pears\nbananas\n",
+        ),
+    ],
+)
+def test_strip_comments1(input_data, mask, expected):
+    """Test strip_comments function"""
+    assert main.strip_comments(input_data, mask) != expected
+
+
+def test_strip_comments_except():
+    """Test strip_comments function raises exception"""
+    with pytest.raises(TypeError):
+        main.strip_comments(int)
```

Comment:
`strip_comments` should have 2 args, or not?

Reply:
Yeap, when we call func with missing args we will get TypeError